### PR TITLE
unicode * is toxic

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

**Comment by mxcl** [comment]
failure is because LDFLAGS hasn't got our rpath in it. this is odd because we forcibly insert it with our cc shims

**Comment by jhheider** [comment]
explicit gcc means no shims, right?

**Comment by jhheider** [comment]
ohhh, right. rpath. grr.

**Comment by jhheider** [comment]
good error: `building for macOS-x86_64 but attempting to link with file built for macOS-x86_64`

File: projects/postgresql.org/package.yml [hunk 1]
```diff
@@ -40,8 +40,18 @@ build:
...
       - --with-libxslt
+    LDFLAGS:
+      - -Wl,-rpath,{{pkgx.prefix}}
     darwin:
-      LDFLAGS: -headerpad_max_install_names
+      LDFLAGS:
+        - -headerpad_max_install_names
+    linux/x86-64:
+      CFLAGS:
+        - -fPIC
```
Comment (mxcl) [inline projects/postgresql.org/package.yml]: we add these in brewkit

File: projects/postgresql.org/package.yml [hunk 2]
```diff
@@ -40,8 +40,18 @@ build:
       - --with-lz4
       - --with-libxml
       - --with-libxslt
+    LDFLAGS:
+      - -Wl,-rpath,{{pkgx.prefix}}
```
Comment (mxcl) [inline projects/postgresql.org/package.yml]: added to brewkit 0.55.1

**Comment by mxcl** [comment]
@jhheider hmm not good https://github.com/pkgxdev/pantry/actions/runs/6877165192/job/18704274799?pr=4049#step:7:2121

**Comment by mxcl** [comment]
nice tricks. No clue how you figured that out!

**Comment by jhheider** [comment]
it's always some set of the same tricks. i think tools get confused calling clang as gcc. though no one should.

**Comment by Bhagirath65** [comment]
nice

**Review by Yakin15437 — COMMENTED** [review]
Good

**Pull request merged**